uldlib/frontend.py

```diff
@@ -265,3 +266,146 @@ def _loop(self, info: DownloadInfo, parts: List[DownloadPart], stop_event: threa
             str(timedelta(seconds=round(elapsed))),
             round(speed / 1024**2, 2)
         ))
+
+class JsonFrontend(Frontend):
```

Comment:
I would rather name it `JSONFrontend`, JSON is an abbreviation.

Reply:
Agree and done in https://github.com/setnicka/ulozto-downloader/pull/132/commits/1778a6aa95ce028386fc326e09c5e17d0aa1e581